tests/xtro-sharpie/EnumCheck.cs

```diff
@@ -131,12 +132,14 @@ public override void VisitEnumDecl (EnumDecl decl, VisitKind visitKind)
 			}
 
 			// check correct [Native] decoration
-			if (native) {
-				if (!IsNative (type))
-					Log.On (framework).Add ($"!missing-enum-native! {name}");
-			} else {
-				if (IsNative (type))
-					Log.On (framework).Add ($"!extra-enum-native! {name}");
+			if (!decl.IsDeprecated ()) {
+				if (native) {
+					if (!IsNative (type))
+						Log.On (framework).Add ($"!missing-enum-native! {name}");
```

Comment:
Single if?

Reply:
This only shows up as changed because the code is re-indented (I added another if at the top). If you ignore whitespace in the diff this doesn't show up.